source/clear-linux/guides/maintenance/fix-broken-install.rst

```diff
@@ -0,0 +1,79 @@
+.. _fix-broken-install:
+
+Fix a broken installation of |CL-ATTR|
+######################################
+
+This guide explains how to fix a broken installation of |CL| using a live
+desktop image on a USB. It's assumed you already installed |CL| on a
+target system, but your operating system does not boot or function properly.
+This guide addresses installation issues that may have impacted the
+:file:`/usr` directory.
+
+Prerequisites
+*************
+
+* :ref:`Download and install the live desktop image on a USB <bare-metal-install-beta>`
+
+Boot a live desktop image to fix target system
+**********************************************
+
+#. Boot the |CL| live desktop image.
+
+.. include:: ../../get-started/bare-metal-install-beta/bare-metal-install-beta.rst
+   :start-after: install-on-target-start:
+   :end-before: install-on-target-end:
+
+Mount root, verify, and run swupd
+*********************************
+
+#. Open a Terminal window.
+
+#. Log in as `root` and set a password.
+
+#. Assure the system is connected to the network.
+
+#. Mount the system’s root partition.
+
+   #. To find the root partition, run:
+
+      .. code-block:: bash
+
+         lsblk
+
+      We find the root partition at `/dev/sda3/`.
```

Comment:
Change "We find the root partition at `/dev/sda3/`." --> "We will use /dev/sda3 as the example root partition."